### ISSUE TITLE
add favicon

### ISSUE THREAD

**Issue by bukinoshita** [infinity-co/infinity-site#11]

**Comment by vitordino** [comment]
does any of this fit your needs? (fuchsia is transparency)

![screen shot 2018-12-27 at 10 20 11 pm](https://user-images.githubusercontent.com/5063967/50498186-b2b84800-0a25-11e9-8010-299a8cc929f1.png)

**Comment by bukinoshita** [comment]
this is on hold yet, need to finish the design. Thanks tho

**Comment by bukinoshita** [comment]
blocked by https://github.com/infinity-co/infinity-media/issues/6

**Comment by bukinoshita** [comment]
Not blocked anymore, we do have [infinity favicon](https://github.com/infinity-co/infinity-media/tree/canary/infinity-assets/infinity-favicon) now.
